python/paddle/tensor/math.py

```diff
@@ -967,6 +967,81 @@ def nansum(x, axis=None, dtype=None, keepdim=False, name=None):
     return sum(tmp_tensor, axis, dtype, keepdim, name)
 
 
+def nanmean(x,axis=None,keepdim=None,name=None):
```

Comment:
keepdim=False is better？keep same with other API？

Reply:
thank you for your suggestion! keep same with other API is the better way. i will update this